### DISCUSSION TITLE
cd: not a directory: httpd

### DISCUSSION THREAD

**Discussion by jnohymn** [Homebrew/discussions#4023] [Getting started]
Hi,

I need to edit the httpd.conf in usr/local/bin/httpd but I am unable to navigate there because I get this response:

cd: not a directory: httpd

This is the latest Homebrew code with Ventura.

Brew doctor is everything is A-Okay and I brew services start httpd seems to work -- but useless without editing the httpd.conf

Thanks for any suggestions you may have.

**Comment by SMillerDev** [comment]
That's not a directory, it's a file. You should check `/usr/local/etc` for configuration

**Comment by jnohymn** [comment]
Thank you so much. You are absolutely correct! 

I have the localhost running under homebrew now.
